src/frontend/mosh-server.cc

```diff
@@ -502,6 +517,14 @@ static int run_server( const char *desired_ip, const char *desired_port,
       exit( 1 );
     }
 
+    /* set SSH_AUTH_SOCK */
+    if ( agent.active() ) {
+      if ( setenv( "SSH_AUTH_SOCK", agent.listener_path().c_str(), true ) < 0 ) {
+	perror( "setenv" );
```

Comment:
I think this comment is suggesting we could define a constant `SSH_AUTH_SOCK` to use in the `setenv()` calls instead of a hardcoded string.

```
#define SSH_AUTH_SOCK "SSH_AUTH_SOCK"
```